README.md

```diff
@@ -1,3 +1,25 @@
-# TODO
 
-Use this template when creating new typescript projects.
+# io-pay
+- [io-pay](#io-pay)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  
+Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
+
+From now wont be true _"pagoPA non è un sito dove pagare"_ it'll be too 🚀
```

Comment:
I guess `wont` to be a typo? I understand the essence but not exactly how you want to say it

Reply:
It's typo, but the sentence should be removed too,  it isn't true. `io-pay` at the moment it will be a WISP replacement